SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -401,14 +408,27 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 		return;
 	}
 
-	// If we're the client, we can now ask the server to spawn our controller.
-	if (!IsServer())
+	if (IsServer())
+	{
+		if (GlobalStateManager != nullptr)
+		{
+			GlobalStateManager->SetCanBeginPlay(true);
+			GlobalStateManager->TriggerBeginPlay();
+			GlobalStateManager->SetAcceptingPlayers(true);
+		}
+		else
+		{
+			UE_LOG(LogSpatial, Log, TEXT("Trying to call GlobalStateManager->SetAcceptingPlayers(true) when GlobalStateManager is nullptr"),);
+		}
+	}
+	else
 	{
+		// If we're the client, we can now ask the server to spawn our controller.
```

Comment:
I'm confused by these comments. We're not asking to spawn here, just saying that we're waiting for server travel to be ready? Should these comments be moved to where this bool is actioned?

Reply:
Yeah this is just wrong. Should have updated the comment.